planner/core/stringer_test.go

```diff
@@ -66,11 +66,11 @@ func TestPlanStringer(t *testing.T) {
 		},
 		{
 			sql:  "show tables in test like 't%'",
-			plan: "Show(table_pattern:[(?i)t.*])",
+			plan: "Show(table_pattern:[t%])",
 		},
 		{
 			sql:  "show tables in test like '%T%'",
-			plan: "Show(table_pattern:[(?i).*T.*])",
+			plan: "Show(table_pattern:[%t%])",
```

Comment:
Not sure is this a good way. We can't get `case-insensitive` information from the plan right now.

Reply:
In fact, We need case-insensitivity when to ```show``` executor. so we add case-insensitive push-down to the ```show```.